src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -183,8 +185,8 @@ public boolean installPlugins(final PluginSpec... specs) throws UnableToResolveD
             checkForUpdates();
         }
 
-        LOGGER.info("Installing plugins by direct upload: " + uploadPlugins);
         if (uploadPlugins) {
```

Comment:
It would be interesting to have a way to have both `uploadPlugins=false` and have the MUC disabled. Maybe an additional flag that is checked at the beginning of `mockUpdateCenter.ensureRunning()`

Reply:
@andresrc, Why?

Reply:
I think @andresrc had in mind tests which specifically check behavior of the actual update center.